lms/static/sass/views/_teams.scss

```diff
@@ -0,0 +1,259 @@
+.view-teams {
+
+  // resets for existing styles
```

Comment:
Nicely documented and defined! One question, is it worth moving these into their own sass partial to be used in other views/UI to come? 

It'd be great to have something we could import to reset things that we know are common. I could see another example view like the following:

```scss
.view-dashboard {
     // resets for existing styles that conflict with this view/UI
     @import shame-reset;
     ... (local resets) ...

    ... (sweet sweet styles) ...
}
```

Feel free to abstract that out now if it makes sense and if so, only bring in ones that aren't too specific to this view.